### ISSUE TITLE
enableFindWidget in WebviewPanelOptions has no effect - unable to disable Find widget

### ISSUE THREAD

**Issue by kdcro101** [Microsoft/vscode#60807]
- VSCode Version: `1.28.1`
- OS Version: `x86_64 GNU/Linux  Debian buster/sid` 
- Tested with latest Insiders edition: `Yes`

Steps to Reproduce:

1. Create panel:
```ts
const panel = vscode.window.createWebviewPanel("my-panel", "Panel title", vscode.ViewColumn.Active, {
            enableScripts: true,
            enableFindWidget: false,
            localResourceRoots: [...],
            retainContextWhenHidden: true,
        });

panel.reveal();
```

2. Focus on panel
3. Press `Ctrl+F`

It is impossible to disable `Find widget` even when explicitly defining `enableFindWidget` as `false` (it should be disabled by default as per vscode.d.ts `WebviewPanelOptions` documentation)

Does this issue occur when all extensions are disabled?: `Yes`

**Comment by johnspurlock** [comment]
This is still occurring in the latest Insiders version: 1.31.0-insider 2019-01-31T06:07:23.930Z 7f136b44cff0cd3486ac88b43598b8c72b025d16

`enableFindWidget` has no effect.

**Comment by johnspurlock** [comment]
Thanks!  Confirmed fixed in Version: 1.32.0-insider
Commit: aeaef41d51201e555735f5e8d2f38a9d0ddb9026
Date: 2019-02-20T06:14:30.644Z
